kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaTopicManager.java

```diff
@@ -316,9 +312,7 @@ private static void removePersistentTopicAndReferenceProducer(final String topic
                             topicName, (producer.getTopic() == null) ? "null" : producer.getTopic().getName());
                 }
             }
-        } catch (InterruptedException | ExecutionException e) {
-            log.error("Failed to get topic '{}' in removeTopicAndReferenceProducer", topicName, e);
-        }
+        });
```

Comment:
Do we need to handle the exceptionally? I see topicFuture might complete exceptionally.

Reply:
Addressed.